Sources/XCTest/XCTestSuite.swift

```diff
@@ -10,28 +10,15 @@
 //  XCTestSuite.swift
 //  A collection of test cases.
 //
+// A concrete subclass of XCTest, XCTestSuite is a collection of test cases.
+// Suites are usually managed by the IDE, but XCTestSuite can also be
```

Comment:
This is not true in the case of swift-corelibs-xctest. We build up a hierarchy of suites based on the test cases the user passes into `XCTMain()`. No IDE (Xcode, in the case of Apple XCTest) is involved.